src/server/middleware/security/robot.go

```diff
@@ -46,7 +46,7 @@ func (r *robot) Generate(req *http.Request) security.Context {
 	}
 	rtk, err := pkg_token.Parse(defaultOpt, robotTk, rClaims)
 	if err != nil {
-		log.Errorf("failed to decrypt robot token: %v", err)
+		log.Debugf("failed to decrypt robot token: %v", err)
```

Comment:
You should include the robot name in the msg for better clarity.

Reply:
added